docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma_2/cancer_model.rst

```diff
@@ -457,71 +457,11 @@ Model Assumptions and Limitations
-+++++++++++++++++++
-
- - Model 1 (Susceptible to MM): compare simulation baseline results of MM prevalence, 
-   MM incidence, and MM cause-specific mortality stratified by age, sex, and year to 
-   GBD 2019 age-/sex-specific MM estimates.
- - Model 2 (MM to MM_{Nth}_relapse): compare simulation baseline results of overall 
-   survival and progression-free survival by disease state to line-specific survival 
-   outcomes obtained from [Braunlin-et-al-2020]_.
-
-.. list-table:: Count measures from simulation stratified by disease state and time
-   :widths: 1 10
-   :header-rows: 1
-   
-   * - Measure
-     - Definition
-   * - disease_state
-     - indication of health status 
-   * - t_start
-     - start time since entrance of specified disease state (months)
-   * - t_end
-     - end time since entrance of specified disease state (months)
-   * - deaths
-     - count of deaths among simulants with specified disease state for a given 
-       period of (t_end - t_start) months
-   * - progression
-     - count of incident cases to new line of treatment among simulants with 
-       specified disease state for a given period of (t_end - t_start) months
-   * - person_time
-     - count of person time among simulants with specified disease state contributed 
-       to a given period of (t_end - t_start) months
-
-.. list-table:: OS and PFS from simulation stratified by line of treatment
-   :header-rows: 1
-
-   * - state
-     - line_of_tx
-     - outcome
-     - measure
-     - numerator
-     - denominator
-   * - MM
-     - first
-     - OS
-     - excess mortality
-     - mm_deaths_count
-     - mm_state_person_time
-   * - MM
-     - first
-     - PFS
-     - progression
-     - mm_to_mm_first_relapse_incidence_count
-     - mm_state_person_time
-   * - MM_{Nth}_relapse
-     - N+1
-     - OS
-     - excess mortality
-     - mm_{Nth}_relapse_deaths_count
-     - mm_{Nth}_relapse_state_person_time
-   * - MM_{Nth}_relapse
-     - N+1
-     - PFS
-     - progression
-     - mm_{Nth}_relapse_to_mm_{(N+1)th}_relapse_incidence_count
-     - mm_{Nth}_relapse_state_person_time
-
-Formula to calculate OS or PFS by line of treatment = 
-:math:`\prod \limits_{t=0}^{t<=n} (1 - \frac{numerator}{denominator} \times duration)`
+Validation and Verification Criteria
+++++++++++++++++++++++++++++++++++++
+
+ - MM incidence stratified by age, sex, and year should match GBD 2019 age-/sex-specific MM incidence. MM incidence stratified by only age and year should match GBD 2019 age-specific MM incidence. MM incidence stratified by only sex and year should match GBD 2019 sex-specific MM incidence, among the 30+ year old population (to avoid cohort effects). MM incidence stratified by only year should match GBD 2019 MM incidence, again among the 30+ year old population.
+ - The same as the previous for MM prevalence, except that it only need **approximately** match. As long as deviations are not too large among the largest age groups, this is acceptable.
```

Comment:
How large is too large?  ±10%

Reply:
In Phase 1 we were off by much more than this -- I don't have exact numbers handy, but in the ballpark of 2x GBD in 75-79 year olds and at least 20x in the oldest age group (see the graphs [here](https://github.com/ihmeuw/vivarium_research_multiple_myeloma/blob/b1d2d1f6263a902c17cc3e300c50dee5907b938c/verification/model_0/mm_cause_vs_gbd.ipynb)).

That's what made me unsure how to write this part; we did compare our results with GBD in Phase 1 but our bar was clearly quite low for how well they should match. I don't know how to specify what that bar is/was.